DTS-STN/canadian-dental-care-plan
# Remove `+` from protected routes

### Description
Per title

### Checklist

### Test Instructions
Run application and go to `/en/home`. Click around the features and verify routes aren't broken.
